src/amber/cli/templates/app/README.md.ecr

```diff
@@ -4,14 +4,13 @@ Your description here
 
 ## Installation
 
-Create a pg database called `<%= @name %>_development` and configure the
-`config/database.yml` to provide the credentials to access the table.
+Configure the `config/environments/development.yml` and set the `database_url` with your credentials to your database.
 
 Then:
 
 ```shellsession
 shards update
-amber migrate up
+amber db drop create migrate
```

Comment:
@drujensen Is this valid for sqlite too?

Reply:
Actually, it's incorrect for all db's.  I will remove the `drop`.  

If a model or migration is not created, all three should fail gracefully.